CONTRIBUTORS_GUIDE.md

```diff
@@ -0,0 +1,48 @@
+# Contributors Guide
+
+This guide aims to guide newcomers to getting started with the project standards.
+
+
+## Code Style
+
+For this project we use basically the default configuration for most used IDEs.
+For the configurations below, make sure to properly configure your IDE:
+
+- **imports**: goimports
+
+For IntelliJ you can find an example `editorconfig` file [here](contrib/intellij.editorconfig).
```

Comment:
Is there a link to the IntelliJ/Goland web site where the contributor can look for to configure their IDE?

Reply:
yes :)